exercises/complex-numbers/description.md

```diff
@@ -1,29 +1,95 @@
 # Description
 
-A complex number is a number in the form `a + b * i` where `a` and `b` are real and `i` satisfies `i^2 = -1`.
+A **complex number** is expressed in the form `z = a + b * i`, where:
 
-`a` is called the real part and `b` is called the imaginary part of `z`.
-The conjugate of the number `a + b * i` is the number `a - b * i`.
-The absolute value of a complex number `z = a + b * i` is a real number `|z| = sqrt(a^2 + b^2)`. The square of the absolute value `|z|^2` is the result of multiplication of `z` by its complex conjugate.
+- `a` is the **real part** (a real number),
 
-The sum/difference of two complex numbers involves adding/subtracting their real and imaginary parts separately:
-`(a + i * b) + (c + i * d) = (a + c) + (b + d) * i`,
-`(a + i * b) - (c + i * d) = (a - c) + (b - d) * i`.
+- `b` is the **imaginary part** (also a real number), and
 
-Multiplication result is by definition
-`(a + i * b) * (c + i * d) = (a * c - b * d) + (b * c + a * d) * i`.
+- `i` is the **imaginary unit** satisfying `i^2 = -1`.
 
-The reciprocal of a non-zero complex number is
-`1 / (a + i * b) = a/(a^2 + b^2) - b/(a^2 + b^2) * i`.
+## Key Properties
 
-Dividing a complex number `a + i * b` by another `c + i * d` gives:
-`(a + i * b) / (c + i * d) = (a * c + b * d)/(c^2 + d^2) + (b * c - a * d)/(c^2 + d^2) * i`.
+### Conjugate
 
-Raising e to a complex exponent can be expressed as `e^(a + i * b) = e^a * e^(i * b)`, the last term of which is given by Euler's formula `e^(i * b) = cos(b) + i * sin(b)`.
+The conjugate of the complex number `z = a + b * i` is given by:
 
-Implement the following operations:
+```text
+z̅ = a - b * i
+```
 
-- addition, subtraction, multiplication and division of two complex numbers,
-- conjugate, absolute value, exponent of a given complex number.
+### Absolute Value
 
-Assume the programming language you are using does not have an implementation of complex numbers.
+The absolute value (or modulus) of `z` is defined as:
+
+```text
+|z| = sqrt(a^2 + b^2)
```

Comment:
Should these use superscript 2 and not the `^` symbol, in like with the other codeblocks?

Reply:
The exponentiation still uses the `^` symbol. It might be better to avoid superscripts and subscripts altogether, as they can be a bit too small for some people to read.

Reply:
Consistency over "too small to read" because I can read them with a reader, or make the font size bigger, but being inconsistent makes things "jitter" a bit in what has to be understood.

Reply:
Completely agree. Removing the superscripts and subscripts means that we revert back to how the instructions currently are. So a^2, b^2, z1, z2, e^a etc.

Reply:
Sure, I am happy to change it to superscript!

Reply:
Superscripts and subscripts aren't easy for everyone to read. Some people may have low-quality monitors, while others may have impaired eyesight. They pose accessibility challenges, as they are small compared to normal text and disrupt visual alignment, making it harder to follow the content. This creates a similar 'jitter' effect that Kotp mentioned. If Exercism were to implement proper math blocks in the future, subscript and superscript notation would be a great option. For now, they are just a workaround due to the lack of proper math rendering.

Please revert everything to how it was before, as it wasn't an issue previously.

Reply:
@tasxatzial I haven’t made any changes yet; I’m still waiting for a conclusion to be drawn!

Reply:
There are two options. Either switch to how it was before, or use superscripts/subscripts. Both of them have no consistency issues. And since there are issues with superscripts/subscripts, it's probably best if we avoid them altogether.

Also, take a look at the complex conjugate `z̅ = a - b * i`. What is the first symbol? A `z`? The line above z is almost touching the letter, making it illegible. The current instructions don't even use a special notation for the conjugate.

All those issues wouldn't exist if Exercism properly supported math.

Reply:
We can also represent the conjugate as `z*` if the bar notation causes issues, or we can avoid denoting it altogether. However, I believe this representation should work fine.  

I’m also happy to revert to the original notation and avoid using subscript or superscript if no one has any concerns with that.

Reply:
> However, I believe the `z*` representation should be fine.

This now looks like the multiplication symbol. I'd just assign the conjugate to a variable and use it:

The conjugate `zc` of the complex number `z = a + b * i` is given by...

Reply:
It sounds like people are leaning towards using the `^` symbol. I don't have a strong opinion here. Using `^` sounds like the most consistent option.

Reply:
Sure, we’ll use the `^` symbol and avoid using notation for the conjugate altogether to prevent issues with bar rendering and the potential confusion of `z*` with multiplication.